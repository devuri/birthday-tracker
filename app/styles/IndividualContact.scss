individualContact {
  align-items: center;
  background-color: #09C6AB;
  flex: 1;
}

notesField {
  width: 250px;
}

individualAvatar {
  border-radius: 62.5px;
  height: 125px;
  margin-top: 75px;
  width: 125px;
}
