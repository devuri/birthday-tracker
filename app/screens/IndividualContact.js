'use strict';
import React, { Component } from 'react';
import styles from '../styles/main';
import { View, Text } from 'react-native';

export default class Contact extends Component {
  render() {
    const contactInfo = this.dataSource.cloneWithRows(this.props.contacts);
    return (
      <View>
        <Text>Hello</Text>
        dataSource={dataSource}
      </View>
    );
  }
}
