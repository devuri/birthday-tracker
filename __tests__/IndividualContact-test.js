import 'react-native';
import React from 'react';
import IndividualContact from '../app/screens/IndividualContact.js';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <IndividualContact />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
