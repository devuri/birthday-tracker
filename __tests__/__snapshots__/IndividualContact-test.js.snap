exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#09C6AB",
      "flex": 1,
    }
  }>
  <View>
    <Image
      source={1}
      style={
        Object {
          "borderRadius": 62,
          "height": 125,
          "marginBottom": 15,
          "marginLeft": 0,
          "marginRight": 0,
          "marginTop": 75,
          "width": 125,
        }
      } />
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#013240",
        "fontSize": 16,
        "fontWeight": "bold",
        "paddingBottom": 5,
        "paddingLeft": 20,
        "paddingRight": 0,
        "paddingTop": 0,
      }
    }>
    undefined undefined
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#013240",
      }
    }>
    🎈
  </Text>
  <View
    style={
      Object {
        "width": 250,
      }
    }>
    <ScrollView>
      <View
        style={
          Array [
            Object {
              "height": 72,
              "paddingBottom": 7,
              "paddingTop": 30,
              "position": "relative",
            },
            undefined,
          ]
        }>
        <TextInput
          keyboardType="default"
          onBlur={[Function]}
          onChangeText={[Function]}
          onFocus={[Function]}
          style={
            Array [
              Object {
                "fontSize": 16,
                "height": 34,
                "lineHeight": 34,
              },
              Object {
                "color": "#013240",
              },
              Object {},
              Object {},
              undefined,
            ]
          }
          underlineColorAndroid="rgba(0,0,0,0)"
          value="" />
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "height": 1,
              },
              Object {
                "backgroundColor": "#EAE8E8",
              },
            ]
          }>
          <View
            style={
              Object {
                "backgroundColor": "#02556D",
                "height": 1,
                "width": 0,
              }
            } />
        </View>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          onPress={[Function]}
          style={
            Object {
              "backgroundColor": "rgba(0,0,0,0)",
              "color": "#02556D",
              "fontSize": 16,
              "left": 0,
              "position": "absolute",
              "top": 37,
            }
          }>
          Present ideas...
        </Text>
      </View>
    </ScrollView>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }>
    <View
      style={
        Object {
          "backgroundColor": "#F7E74A",
          "borderRadius": 5,
          "margin": 10,
          "padding": 5,
          "shadowColor": "#000000",
          "shadowOpacity": 0.3,
          "shadowRadius": 3,
          "width": 80,
        }
      }>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            undefined,
          ]
        }
        testID={undefined}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#02556D",
              "fontSize": 18,
              "textAlign": "center",
            }
          }>
          Edit
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "backgroundColor": "#F7E74A",
          "borderRadius": 5,
          "margin": 10,
          "padding": 5,
          "shadowColor": "#000000",
          "shadowOpacity": 0.3,
          "shadowRadius": 3,
          "width": 80,
        }
      }>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        hitSlop={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
            },
            undefined,
          ]
        }
        testID={undefined}>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#02556D",
              "fontSize": 18,
              "textAlign": "center",
            }
          }>
          Delete
        </Text>
      </View>
    </View>
  </View>
</View>
`;
